app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -1235,15 +1235,59 @@ a.mention-group {
   }
 }
 
-.paused-animated-image {
+.pausable-animated-image {
   position: relative;
-  display: block;
-  > canvas {
+  display: inline-block;
+
+  > canvas,
+  > .animated-image-overlay {
     position: absolute;
-    top: 0;
-    left: 0;
+    bottom: 0;
+    right: 0;
+  }
+
+  > canvas {
+    background-color: var(--primary-very-low);
+  }
+
+  > .animated-image-overlay {
+    pointer-events: none;
+    text-align: right;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    > .d-icon {
+      cursor: pointer;
+      padding: 0.5em;
+      margin: 0.5em;
+      background-color: rgba(var(--always-black-rgb), 0.25);
+      color: var(--secondary-or-primary);
+      cursor: pointer;
+      display: none;
+    }
+  }
+
+  img.animated {
+    cursor: pointer;
+  }
+
+  html.no-touch
```

Comment:
Am I missing something here? Indentation seems weird for what seems like a top level selector.

Reply:
I suspect it's because the selector is quite long (this was Prettier's doing anyway).